src/base58.cpp

```diff
@@ -12,7 +12,7 @@
 
 /** All alphanumeric characters except for "0", "I", "O", and "l" */
 static const char* pszBase58 = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
-static const int8_t mapBase58[] = {
+static constexpr std::array<int8_t, 256> mapBase58 {
```

Comment:
missing #include <array> ?

Reply:
sorry, I don't familiar with c++11, so just change it  back to old school style.

Reply:
IMHO,  the `.size()` method was worth the import... but anyway.